Mobile Buy SDK/Mobile Buy SDK/Utils/BUYApplePayAuthorizationDelegate.m

```diff
@@ -24,67 +24,44 @@
 //  THE SOFTWARE.
 //
 
-#import "BUYApplePayHelpers.h"
-#import "BUYAddress.h"
+#import "BUYApplePayAuthorizationDelegate.h"
 #import "BUYApplePayAdditions.h"
+#import "BUYApplePayToken.h"
 #import "BUYClient+Checkout.h"
 #import "BUYClient+Storefront.h"
 #import "BUYCheckout.h"
 #import "BUYError.h"
-#import "BUYModelManager.h"
+#import "BUYModelManager+ApplePay.h"
 #import "BUYShop.h"
 #import "BUYShopifyErrorCodes.h"
-#import "BUYApplePayToken.h"
 
 const NSTimeInterval PollDelay = 0.5;
 
-@interface BUYApplePayHelpers ()
-@property (nonatomic, strong) BUYCheckout *checkout;
-@property (nonatomic, strong) BUYClient *client;
+@interface BUYApplePayAuthorizationDelegate ()
 
 @property (nonatomic, strong) NSArray *shippingRates;
 @property (nonatomic, strong) NSError *lastError;
 
-@property (nonatomic, strong) BUYShop *shop;
-
 @end
 
-@implementation BUYApplePayHelpers
-
-- (instancetype)initWithClient:(BUYClient *)client checkout:(BUYCheckout *)checkout
-{
-	return [self initWithClient:client checkout:checkout shop:nil];
-}
+@implementation BUYApplePayAuthorizationDelegate
 
-- (instancetype)initWithClient:(BUYClient *)client checkout:(BUYCheckout *)checkout shop:(BUYShop *)shop
+- (instancetype)initWithClient:(BUYClient *)client checkout:(BUYCheckout *)checkout shopName:(NSString *)shopName
 {
 	NSParameterAssert(client);
 	NSParameterAssert(checkout);
```

Comment:
If "warnings as errors" are disabled nullability will simply be a warning and isn't enough, in my opinion.